Formula/s/sonarqube.rb

```diff
@@ -1,8 +1,8 @@
 class Sonarqube < Formula
   desc "Manage code quality"
   homepage "https://www.sonarsource.com/products/sonarqube/"
-  url "https://binaries.sonarsource.com/Distribution/sonarqube/sonarqube-10.7.0.96327.zip"
-  sha256 "4a5e5f0d7581e287cff4f02f7588dae499e2aa81fce37b88612acadfb4ba4518"
+  url "https://binaries.sonarsource.com/Distribution/sonarqube/sonarqube-24.12.0.100206.zip"
```

Comment:
That's a pretty significant bump, but it is what's available upstream.

Reply:
Still waiting for a release description https://github.com/SonarSource/sonarqube/releases/tag/24.12.0.100206
or announcement https://www.sonarsource.com/products/sonarqube/whats-new/

Reply:
The switch to CalVer is confirmed here: https://community.sonarsource.com/t/sonarqube-community-build-24-12-0-100206-released/131515